front/components/items/Appointment/helper.ts

```diff
@@ -1,21 +1,21 @@
-import { strapiAppointmentQuery } from 'lib/strapi/appointments/queryType';
+import { propsAppointment, propsOfFormingDate, propsOfGetId } from './models';
 
-export type filters = {
-    [Keys: string]: Object;
-};
-type props = {
-    appointments: strapiAppointmentQuery[] | undefined;
+export const formingDate = ({ chosenDate }: propsOfFormingDate) => {
+    // changing timezone by reducing by 2h (because of polish timezone (GMT+2))
+    const rightTime = new Date(chosenDate?.getTime() - 60 * 1000 * -120);
+    // changing format of date to YYYY-MM-DD
+    return rightTime?.toISOString().slice(0, 10);
 };
 
-export const getTimeOfAppointments = ({ appointments }: props) => {
+export const getTimeOfAppointments = ({ appointments }: propsAppointment) => {
     let allHours = appointments?.map((item) => item.attributes.time);
     let sortedHours = allHours?.sort(function (a, b) {
```

Comment:
```suggestion
    let sortedHours = allHours?.sort((a, b) => Number(a.replace(/:/g, '')) - Number(b.replace(/:/g, ''));
```

keep one convention. One line above you used arrow function, rn you are using name function